readme.md

```diff
@@ -168,6 +168,7 @@ Here's a list of all the icons currently supported. Feel free to open an issue t
 |      `gmail`       |     <img src="./icons/Gmail-Dark.svg" width="48">     |
 |     `gherkin`      |    <img src="./icons/Gherkin-Dark.svg" width="48">    |
 |        `go`        |       <img src="./icons/GoLang.svg" width="48">       |
+|        `goland`        |       <img src="./icons/GoLand-Dark.svg" width="48">       |
```

Comment:
Could you please remove extra spaces before `goland` in order make proper format and match '|' with above '|'?

Reply:
done